src/service.js

```diff
@@ -64,11 +64,14 @@ async function fetchBlock(dispatcher, cid) {
     return null
   }
 
-  const separator = car.indexOf('/')
-  const bucket = car.slice(0, separator)
-  const key = car.slice(separator + 1)
+  const bucketSeparator = car.indexOf('/')
```

Comment:
Rather than doing this, I would use the same technique we use here:

https://github.com/web3-storage/ipfs-elastic-provider-indexer-lambda/blob/main/src/index.js#L108

Reply:
Good one!